hawkular-wildfly-agent/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This is where the WF build plugins (feature pack and provisioning) expect the fragment with the agent subsystem's configuration. It should not be empty, I just forgot about it when I was asking your what the default config should be. The content from `hawkular-wildfly-agent-wf-extension/src/main/resources/system/layers/base/org/hawkular/agent/main/subsystem-snippet.xml` should basically be moved here and the Installer should be somehow adapted. I will polish this part later.

The objective of the present PR is to get the Agent working on WF10, Artemis and all the changed upstream components.